### ISSUE TITLE
Failed to load the ruby-quick-test package

### ISSUE THREAD

**Issue by ghostlurker** [philnash/ruby-quick-test#13]
[Enter steps to reproduce below:]

1. ...Start Atom
2. ...Error message appears

**Atom Version**: 0.194.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.3
**Thrown From**: [ruby-quick-test](https://github.com/philnash/ruby-quick-test) package, v0.3.0

### Stack Trace

Failed to load the ruby-quick-test package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
    at __extends (/Users/swidup/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:5:199)
    at /Users/swidup/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:12:5
    at Object.<anonymous> (/Users/swidup/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:133:5)
    at Object.<anonymous> (/Users/swidup/.atom/packages/ruby-quick-test/lib/ruby-quick-test-view.coffee:135:4)
    at Module._compile (module.js:452:26)
    at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
    at Module.load (module.js:347:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at Object.<anonymous> (/Users/swidup/.atom/packages/ruby-quick-test/lib/ruby-quick-test.coffee:4:23)
    at Object.<anonymous> (/Users/swidup/.atom/packages/ruby-quick-test/lib/ruby-quick-test.coffee:21:4)
    at Module._compile (module.js:452:26)
    at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
    at Module.load (module.js:347:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at Package.module.exports.Package.requireMainModule (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:677:34)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:170:28
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
    at Package.module.exports.Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:161:12)
    at PackageManager.module.exports.PackageManager.loadPackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:354:14)
    at PackageManager.module.exports.PackageManager.loadPackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:324:14)
    at Atom.module.exports.Atom.startEditorWindow (/Applications/Atom.app/Contents/Resources/app.asar/src/atom.js:620:21)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:12:8)
    at Object.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-bootstrap.js:23:4)
    at Module._compile (module.js:452:26)
    at Object.loadFile [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
    at Module.load (module.js:347:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at window.onload (file:///Applications/Atom.app/Contents/Resources/app.asar/static/index.js:54:25)
```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "tabs",
      "tree-view",
      "vim-mode",
      "RubyMotion"
    ]
  }
}
```

### Installed Packages

```coffee
# User
Sublime-Style-Column-Selection, v1.2.3
atom-html-preview, v0.1.6
atom-rails, v0.4.0
bug-report, v0.6.3
file-watcher, v0.2.1
git-tab-status, v1.7.0
html-helper, v0.2.3
jsonlint, v1.1.0
language-apache, v1.3.0
language-chef, v0.4.0
language-nginx, v0.5.2
language-rpm, v1.0.0
linter-foodcritic, v0.3.7
linter-js-yaml, v1.0.0
linter-jsonlint, v0.1.2
linter-rubocop, v0.2.3
linter-ruby, v0.1.4
linter-shellcheck, v0.0.7
linter-xmllint, v0.0.5
markdown-format, v2.5.0
markdown-helpers, v0.2.2
markdown-pdf, v1.3.5
markdown-table-formatter, v2.5.2
minimap, v4.7.6
pretty-json, v0.3.3
ruby-block, v0.3.3
ruby-bundler, v0.2.1
ruby-define-method, v0.2.0
ruby-quick-test, v0.3.0
sublime-tabs, v0.5.6
tabs-to-spaces, v0.9.2
tag, v0.2.3
text-manipulation, v0.6.0
trailing-spaces, v0.2.5

# Dev
No dev packages
```

**Comment by philnash** [comment]
Well that's terrifying. I haven't used Atom for a while I'm afraid. So this is going to take me a while to fix. Any help appreciated.

**Comment by ghostlurker** [comment]
Wish I could, but I'm blocked from any open source contributions by my employer.  Issues are as far as I can take things unfortunately.

**Comment by philnash** [comment]
This has now been fixed with version 0.4.0 of Ruby Quick Test. Thanks for reporting and sorry it took a while!
